### ISSUE TITLE
Error "No such remote: origin" during backporting

### ISSUE THREAD

**Issue by flash1293** [sqren/backport#233]
I just got a weird error while backporting:
```
2020-10-22 17:14:34: exec error 'git remote rm origin':
Error: Command failed: git remote rm origin
fatal: No such remote: 'origin'

    at ChildProcess.exithandler (child_process.js:294:12)
    at ChildProcess.emit (events.js:198:13)
    at maybeClose (internal/child_process.js:982:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
```

I re-ran the same backport without changing anything and it went through just fine:
```
➜  yellow git:(lens/suggestion-keys) ✗ node scripts/backport.js
? Select commit #81466 (ea229891)
? Select branch 7.x

Backporting to 7.x:
✔ Pulling latest changes
✔ Cherry-picking: fix flaky unit test (#81466)
✖ Pushing branch "flash1293:backport/7.x/pr-81466"
An unhandled error occurred. Please consult the logs
➜  yellow git:(lens/suggestion-keys) ✗ node scripts/backport.js
? Select commit #81466 (ea229891)
? Select branch 7.x

Backporting to 7.x:
✔ Pulling latest changes
✔ Cherry-picking: fix flaky unit test (#81466)
✔ Pushing branch "flash1293:backport/7.x/pr-81466"
✔ Creating pull request
✔ Adding labels: backport
View pull request: https://github.com/elastic/kibana/pull/81496
```

I used the same instance to do backports successfully before, nothing changed to my knowledge since the last usage. Maybe some kind of race condition?

**Comment by sqren** [comment]
Interesting. That error should have been swallowed:
https://github.com/sqren/backport/blob/c8777ff9a7c8e9076f4a733e1d1f09a6d2141aa0/src/services/git.ts#L77-L90

Context: the `origin` remote is delete automatically to avoid confusion what it points to. But sometimes it doesn't exist in the first place, which is also not a problem the user should be warned about.

**Comment by jfsiii** [comment]
I'm also experiencing this error, but re-running does not succeed.

I added `console.log('orig e', e);` in the `catch` block on line 84 above (`dist/services/git.js` line 72) and it yields:

```
$ node scripts/backport --verbose --pr  87196
? Select pull request Add the existing route to openapi schema (#87196)
? Select branch 7.x
orig e Error: Command failed: git remote rm origin
error: No such remote: 'origin'

    at ChildProcess.exithandler (child_process.js:308:12)
    at ChildProcess.emit (events.js:315:20)
    at maybeClose (internal/child_process.js:1048:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5) {
  killed: false,
  code: 2,
  signal: null,
  cmd: 'git remote rm origin',
  stdout: '',
  stderr: "error: No such remote: 'origin'\n"
}


⚠️  Ouch! An unknown error occured 😿
Error message: Command failed: git remote rm origin
error: No such remote: 'origin'

Please open an issue in https://github.com/sqren/backport/issues or contact me directly on https://twitter.com/sorenlouv
For additional details see the logs: /Users/jfsiii/.backport/backport.log
```

In my case it seems the issue is the `code: 2` when the script expects `code: 128`. 

I'm not sure if something changed with `exec` or if this indicates a different or problematic error.

As I workaround, I removed the ` && e.code === 128` and re-ran and it worked fine.

**Comment by sqren** [comment]
Thanks both. With your help I was able to create this fix: https://github.com/sqren/backport/pull/241.
LMK if that looks good to you.

**Comment by sqren** [comment]
Fixed and released in 5.6.2
